### PR TITLE
fixup! changes default of waitForActiveShards to ALL

### PR DESCRIPTION
**Pull request by chaudum**

### CODE REVIEW

File: core/src/test/java/org/elasticsearch/discovery/ClusterDisruptionIT.java [hunk 1]
```diff
@@ -301,6 +301,8 @@ public void testSendingShardFailure() throws Exception {
             .setSettings(Settings.builder()
                 .put(IndexMetaData.SETTING_NUMBER_OF_SHARDS, 3)
                 .put(IndexMetaData.SETTING_NUMBER_OF_REPLICAS, 2)
+                .put(IndexMetaData.SETTING_AUTO_EXPAND_REPLICAS, false)
+                .put(IndexMetaData.SETTING_WAIT_FOR_ACTIVE_SHARDS.getKey(), 1)
```
Comment (mxm) [inline core/src/test/java/org/elasticsearch/discovery/ClusterDisruptionIT.java]: This doesn't change it to "ALL" but to 1.

Comment (chaudum) [reply core/src/test/java/org/elasticsearch/discovery/ClusterDisruptionIT.java]: this change only fixes a flaky test, which depends on the default of elasticsearch (not the crate default "ALL")

Comment (mxm) [reply core/src/test/java/org/elasticsearch/discovery/ClusterDisruptionIT.java]: Would be helpful to have this reflected in the PR description or commit message. Context is always good when reviewing a PR :)

Comment (chaudum) [reply core/src/test/java/org/elasticsearch/discovery/ClusterDisruptionIT.java]: yes you're right. should have at least added the commit (6bde5f9fd7) for which the fixup is intended

**Pull request merged**